runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerPlugin.java

```diff
@@ -927,6 +938,18 @@ private boolean processHerbiboarHerbSackLoot(int timestamp)
 		return true;
 	}
 
+	private boolean processMasterFarmerSeedBoxLoot(Matcher matcher)
+	{
+		int quantityStolen = Integer.parseInt(matcher.group(2));
+		List<ItemStack> seeds = new ArrayList<>();
```

Comment:
You can just use `Collections.singletonList()` instead of making a list for one thing.